deploy/helm/kubecf/values.yaml

```diff
@@ -368,10 +260,6 @@ features:
       key: ~
     annotations: {}
     labels: {}
-  suse_default_stack:
-    enabled: false
-  suse_buildpacks:
```

Comment:
Do we need deprecation things for these too? (I'm totally okay with no, but it needed to be asked)

Reply:
We have an entry in `unsupported` for them in https://github.com/cloudfoundry-incubator/kubecf/blob/jandubois/1086-configurable-multi-stack-support/deploy/helm/kubecf/config/sle15.yaml#L3-L5

I think that is good enough; after all this is why i created the `unsupported` mechanism in the first place (was previously called `deprecated`).

Reply:
Indeed; it's just been so long since I looked at it that I forgot; sorry about that!